cv32e20/tests/programs/custom/interrupt_test/interrupt_test.c

```diff
@@ -651,14 +617,56 @@ int test8() {
     return EXIT_SUCCESS;
 }
 
+// Test 7 will repeat the basic interrupt test in test 1
+// But with a relocated vector table via mtvec CSR and DIRECT vector mode
+// CV32E20 does not support DIRECT mode.  Saved here until clean up
+int test7() {
```

Comment:
As far as I can tell, this function is never called.  Nevertheless, it is a integer function that does not return a value which can be a dangerous practise.  I recommend either commenting it out in it's entirety, or returning `EXIT_SUCCESS`.  In the future, if someone mistakenly calls this function, that will minimize the likelihood of unexpected behavior in the callee.

Reply:
I have added TODOs and moved the comment so it comments out the whole function.